src/main/docs/guide/cache-abstraction.adoc

```diff
@@ -74,3 +74,10 @@ Names of caches under `micronaut.caches` should be defined in kebab case (lowerc
 ====
 
 To configure a weigher to be used with the `maximumWeight` configuration, create a bean that implements `com.github.benmanes.caffeine.cache.Weigher`. To associate a given weigher with only a specific cache, annotate the bean with `@Named(<cache name>)`. Weighers without a named qualifier will apply to all caches that don't have a named weigher. If no beans are found, a default implementation will be used.
+
+[NOTE]
+.Native compilation
+====
+When using Caffeine with Native Compilation, the most commonly used caches will be automatically registered.
+If you require additional caches, you will need to register them with Graal yourself https://docs.micronaut.io/latest/guide/#_adding_additional_classes_for_reflective_access[as shown in the guide].
```

Comment:
How are users supposed to know if their cache is registered? Perhaps add a link to the feature class source?

Reply:
The only way I can think of, is to look for ClassNotFound errors in Graal 🤔

Reply:
Right, so then it's maybe a matter of adding a link to `CaffeineFeature.java`